packages/astro/src/vite-plugin-build-html/index.ts

```diff
@@ -46,16 +47,18 @@ export function rollupPluginAstroBuildHTML(options: PluginOptions): VitePlugin {
   // The web path of the src folter
   const srcRootWeb = srcRoot.substr(astroConfig.projectRoot.pathname.length - 1);
 
-  // A map of pages to rendered HTML
-  const renderedPageMap = new Map<string, string>();
-
   //
   const astroScriptMap = new Map<string, string>();
   const astroPageMap = new Map<string, string>();
```

Comment:
I profiled these, as these were suspect too. But all the other maps were pretty lightweight and only contained short strings. In the docs, these all add up to a few hundred kB at most (not the megabytes and megabytes like HTML). So I left them as-is.